pinot-core/src/test/java/org/apache/pinot/queries/DistinctQueriesTest.java

```diff
@@ -245,12 +245,14 @@ public void testSingleColumnDistinctOnlyInnerSegment()
       // String columns
       //@formatter:off
       List<String> queries = Arrays
-          .asList("SELECT DISTINCT(stringColumn) FROM testTable", "SELECT DISTINCT(rawStringColumn) FROM testTable");
+          .asList("SELECT DISTINCT(stringColumn) FROM testTable");
```

Comment:
Revert the change in this file

Reply:
We need the changes since the order of results in absence of ORDER BY is not consistent between segment reads and dictionary reads.

Reply:
I see. This is caused by sorting in alphabetical order. Can you please add some comments explaining the reason?